GitCommands/Settings/ConfigFileSettings.cs

```diff
@@ -70,15 +70,11 @@ public static ConfigFileSettings CreateGlobal(ConfigFileSettings? lowerPriority,
                 SettingLevel.SystemWide);
         }
 
-        public new string GetValue(string setting)
-        {
-            return GetString(setting, string.Empty);
-        }
+        public new string GetValue(string setting) => GetString(setting, string.Empty);
 
-        public IReadOnlyList<string> GetValues(string setting)
-        {
-            return SettingsCache.GetValues(setting);
-        }
+        public bool IsEnabled(string setting) => GetValue(setting).Trim().Equals("true", StringComparison.InvariantCultureIgnoreCase);
+
+        public IReadOnlyList<string> GetValues(string setting) => SettingsCache.GetValues(setting);
```

Comment:
All new public API in here must also be reflected in `IGitModule` interface.

Reply:
... and this surprising method needs an XML doc, (which should not be overused because it easily gets out of sync with the implementation). But in this case, the signature is not self-explaining.

Reply:
This special feature is not used in this PR anyway. Hence I would not expose it.

Reply:
@mstv You are talking about `IReadOnlyList<string> GetValues(string setting)`? It's just a refactor to convert it in an expression-bodied method.

Or you recommend to add xml doc to the new API?

I did both...